code/drasil-printers/lib/Language/Drasil/Markdown/CreateMd.hs

```diff
@@ -97,16 +97,34 @@ license :: Doc -> Doc
 license auth = text "Copyright (c) 2021," <+> auth <>
   text ". All rights reserved. Please see the [full license](https://github.com/JacquesCarette/Drasil/blob/4b9ad0a3016fecb3c7a2aa82ab142f9e805b5cc8/LICENSE) for more details."
 -- | Drasil Tree icon. Uses HTML directly to format image since normal markdown doesn't support it.
-drasilImage :: Doc
-drasilImage = alignImage "../../../../drasil-website/WebInfo/images/Icon.png"
+-- Extracts the backslash from the filepath.
+removeSlash :: [Char] -> [Char]
+removeSlash st = [ x | x <- st, x `elem` "//"]
+
+-- Determines the number of slashes within the FilePath of the SampleInput file
+numSlash :: [Char] -> Int
+numSlash [] = 4
+numSlash file = length (removeSlash file)
```

Comment:
Others may disagree with me, but if `removeSlash` and `numSlash` are only going to be used here, might as well add them to the locally defined functions in the `where` statement of `buildPath`.

Reply:
Ok, I will make that change!